mobile/cmp/button/ColAutosizeButton.js

```diff
@@ -27,8 +27,8 @@ export const [ColAutosizeButton, colAutosizeButton] = hoistCmp.withFactory({
     }) {
         gridModel = withDefault(gridModel, useContextModel(GridModel));
 
-        if (!gridModel) {
-            console.error("No GridModel available to ColAutosizeButton. Provide via a 'gridModel' prop, or context.");
+        if (!gridModel?.autosizeEnabled) {
+            console.error("No GridModel available with autosize enabled. Provide via a 'gridModel' prop, or context.");
```

Comment:
Should probably replace this message with the improved message in the desktop version for consistency

Reply:
Agree -- didn't want to fiddle with mobile/desktop inconsistencies at this point -- sticking with the existing mobile pattern